packages/components/organisms/f-content-cards/src/components/ContentCards.js

```diff
@@ -69,19 +94,56 @@ export default {
         const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
         const localeConfig = tenantConfigs[locale];
 
+        const options = {
+            root: null,
+            rootMargin: '0px',
+            threshold: 0.7
+        };
+
+        const observerCallback = entries => {
```

Comment:
Would be good to have a few more comments about what's happening in this method, as there's a lot going on in this `provide()` now.

At a glance, it looks like it's reporting when a card gets intersected with onscreen and provides data on that, but would be good to give a bit of a non-code overview somewhere.

Reply:
your glance is correct, I will add a comment above to indicate what it's doing from a non code perspective

Reply:
I have added an extended comment